src/realm/sync/apply_to_state_command.cpp

```diff
@@ -0,0 +1,407 @@
+#include <algorithm>
+#include <cinttypes>
```

Comment:
We're just including `<stddef.h>` in a couple of other places to avoid having to write `std::size_t`, and can just use `size_t`.